testing/coordinator.go

```diff
@@ -71,57 +71,6 @@ func (coord *Coordinator) UpdateTimeForChain(chain *TestChain) {
 	chain.CurrentHeader.Time = coord.CurrentTime.UTC()
 }
 
-// Setup constructs a TM client, connection, and channel on both chains provided. It will
-// fail if any error occurs. The clientID's, TestConnections, and TestChannels are returned
-// for both chains. The channels created are connected to the ibc-transfer application.
-func (coord *Coordinator) Setup(path *Path) {
```

Comment:
I think maybe we should deprecate these functions? It's a quick change, but will cause enormous diffs when updating to v9 for our users. So might be worth giving them to the option to separate this diff between v9 and v10

We can just add these back calling into path.Setup()